bot/cogs/memes_cog/memes_cog.py

```diff
@@ -6,12 +6,20 @@
 import random
 import time
 
+import io
+
+import os
+import datetime
+from PIL import Image, ImageDraw, ImageSequence, ImageFont
```

Comment:
When importing from a library its always best to be explicit about what is our code and what is library code, so we shouldnt import types directly instead we should call them with the module name or an alias eg 

`import PIL as p` then `p.Image` or `p.ImageDraw` just to be clear about what our code is calling